plugin/activity-log.vim

```diff
@@ -61,10 +61,10 @@ augroup END
 " Log create, interval/break, continue, end task
 "" sets ,ls mapping to log current line as start of task in .log files
 "" sets ,le mapping to log current line as end of task in .log files
-nmap <silent> <Leader>ls :call TimeLogLine('start_task')<CR>
-nmap <silent> <Leader>le :call TimeLogLine('end_task')<CR>
-nmap <silent> <Leader>li :call TimeLogLine('interval_task')<CR>
-nmap <silent> <Leader>lc :call TimeLogLine('continue_task')<CR>
+" nmap <silent> <Leader>ls :call TimeLogLine('start_task')<CR>
+" nmap <silent> <Leader>le :call TimeLogLine('end_task')<CR>
+" nmap <silent> <Leader>li :call TimeLogLine('interval_task')<CR>
+" nmap <silent> <Leader>lc :call TimeLogLine('continue_task')<CR>
```

Comment:
:+1: That should never have been there =) - the equivalent should still work though, and be mentioned in the readme though.